lib/ctu.cpp

```diff
@@ -312,7 +312,8 @@ CTU::FileInfo *CTU::getFileInfo(const Tokenizer *tokenizer)
         for (const Token *tok = scope.bodyStart; tok != scope.bodyEnd; tok = tok->next()) {
             if (tok->str() != "(" || !tok->astOperand1() || !tok->astOperand2())
                 continue;
-            if (!tok->astOperand1()->function())
+            const Function* fct = tok->astOperand1()->function();
```

Comment:
how about clarifying the function pointers. I suggest `scopeFunction` and `tokFunction` ?